docs/data/variant-calls.md

```diff
@@ -0,0 +1,71 @@
+---
+id: variant-calls
+title: Variant Calls
+sidebar_label: Variant Calls
+---
+
+ICGC ARGO provides access to data from ARGO member programs generated through the standardized [ARGO Analysis pipelines](/docs/analysis_workflows/analysis-overview). The following page lists variant calling analyses and data files generated by the ARGO Analysis pipelines.
+
+## Raw SNV Calls
+
+Data files containing Single Nucleotide Variations (SNV) called from aligned reads, which have not yet been annotated but do have some filtering flags added.
```

Comment:
does the user know what "filtering flags" are? asking for a friend haha

Reply:
hahah Christina said yes lol